src/main/java/soolutions/hometaskapp/account/Deposit.java

```diff
@@ -0,0 +1,36 @@
+package soolutions.hometaskapp.account;
+
+import soolutions.hometaskapp.common.Amount;
+
+import java.time.Instant;
+
+public class Deposit implements Transaction {
+    private final Instant occurredAt;
+    private final String description;
+    private final Amount amount;
+
+    public Deposit(Amount amount, String description) {
+        this(amount, description, Instant.now());
+    }
+
+    public Deposit(Amount amount, String description, Instant occurredAt) {
+        this.occurredAt = occurredAt;
+        this.description = description;
+        this.amount = amount;
+    }
+
+    @Override
+    public Instant occurredAt() {
```

Comment:
@SilasReinagel have you read Yegor's take on "printers vs. getters"? Why do we have these getters? Do we need them right now?

Reply:
Unit test won't pass without them. Do you have specific alternative way to make the test pass?